client/router.js

```diff
@@ -206,6 +206,8 @@ ClientRouter.prototype.redirectTo = function(path, options) {
 
   if (options.pushState === false) {
     // Do a full-page redirect.
+    // add a leading slash so path is not appended
+    if (path.charAt(0) !== '/') path = '/' + path;
     window.location.href = path;
   } else {
     // Do a pushState navigation.
```

Comment:
if `pushState === true` this happens.  The weird thing about this flow is that `pushState` is `false` is it's way of saying, 'no route found' to this method.  If you look above to the `navigate` method that checks to see if the route matches it shows the intentions (IMHO of course).

And sure thing I'll add a test.